packages/ckeditor5-dev-utils/src/builds/getdllpluginwebpackconfig.ts

```diff
@@ -1,217 +0,0 @@
-/**
- * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
- * For licensing, see LICENSE.md.
- */
-
-import fs from 'fs';
-import path from 'path';
-import { CKEditorTranslationsPlugin } from '@ckeditor/ckeditor5-dev-translations';
```

Comment:
`@ckeditor/ckeditor5-dev-translations` can be removed from `package.json`.

Also, `packages/ckeditor5-dev-utils/types/ckeditor5-dev-translations.d.ts` seems unused.